doc/local.conf

```diff
@@ -305,6 +312,12 @@ DISTRO_FEATURES_remove = "pam selinux"
 # * "0" - boot update will not be included.
 #BUNDLE_BOOT = "1"
 
+#
```

Comment:
No need to add possibility to override it in local conf. Just change in aos-image-update.bb.

Reply:
Done